modules/nw-assign-ip.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc
+
+[id="nw-assign-ip_{context}"]
+= Assigning IP addresses
+
+To use IP address blocks defined by `autoAssignCIDRs` in {product-title}, you must configure the necessary IP address assignment and routing for your host network.
+
+.Prerequisites
+
+* You installed a cluster on bare metal infrastructure.
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Edit the `network.config.openshift.io` YAML file:
++
+[source,terminal]
+----
+oc edit network.config.openshift.io cluster -o yaml
+----
++
+. Assign IP addresses to `externalIP`:
++
+.Example `network.config.openshift.io` YAML
++
+[source,yaml]
+----
```

Comment:
@bmcelvee, this is always the same object, because it's an OCP config object. I think we should always include the full context, which is:

```
kind: Network
apiVersion: config.openshift.io/v1
metadata:
  name: cluster
# ... rest of the YAML from below
```